src/pages/api/create-pr.ts

```diff
@@ -121,10 +159,55 @@ export function validateRequestBody(body: unknown): ApiError | ApiSuccess<Create
       warpRouteId,
       organization,
       username,
+      deployConfigResult,
+      warpConfigResult,
     },
   };
 }
 
+const MAX_TIMESTAMP_DURATION = 2 * 60 * 1000; // 2 minutes
+
+async function validateRequestSignature(
+  signatureVerification: unknown,
+): Promise<ApiError | ApiSuccess<VerifyPrSignature>> {
+  if (!signatureVerification) return { error: 'Missing signatureVerification' };
+
+  const parsedSignatureBody = VerifyPrSignatureSchema.safeParse(signatureVerification);
+  if (!parsedSignatureBody.success) return { error: zodErrorToString(parsedSignatureBody.error) };
+
+  const { address, message, signature } = parsedSignatureBody.data;
+
+  if (!isHex(address)) return { error: 'Address is not a hex string' };
```

Comment:
Can't we use the utilities from the utils package fro checking that the address is an EVM address instead?